com.unity.cinemachine/Samples~/Cinemachine Example Scenes/Shared/Scripts/ExampleHelpWindow.cs

```diff
@@ -2,43 +2,40 @@
 
 namespace Cinemachine.Examples
 {
-
-[AddComponentMenu("")] // Don't display in add component menu
-public class ExampleHelpWindow : MonoBehaviour
-{
-    public string m_Title;
-    [TextArea(minLines: 10, maxLines: 50)]
-    public string m_Description;
-
```

Comment:
This name change breaks all the samples that use this script.  You should put such a change in the samples upgrade branch, not here.